### PR TITLE
added git workflow update_git_index_after_ignoring_files.yaml

### PR DESCRIPTION
**Pull request by vjindal0112**
## Description of changes (updated or new workflows)

added git workflow `update_git_index_after_ignoring_files.yaml`

It covers the case of regenerating the git index after ignoring files which were previously committed to your repository.

### CODE REVIEW

File: specs/git/update_git_index_after_ignoring_files.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+name: Delete newly git-ignored files from your repository
+# The corresponding command for the workflow. Any arguments should be surrounded with two curly braces. E.g `command {{arg}}`.
+command: |-
+  git rm -r --cached .
+  git add .
+# Any tags that the workflow should be categorized with.
+tags:
+  - git
+  - gitignore
+  - version control
```
Comment (alokedesai) [inline specs/git/update_git_index_after_ignoring_files.yaml:12]: in our experience, the best workflows have fewer tags (see https://github.com/warpdotdev/workflows#what-makes-a-useful-workflow). Given that--could we remove the `gitignore` and `version control` flags?

**Review by alokedesai — CHANGES_REQUESTED** [review]
Thank you so much for contributing--this looks great! Just have one comment about the tags

File: specs/git/update_git_index_after_ignoring_files.yaml (hunk 1 above)
Comment (vjindal0112) [reply specs/git/update_git_index_after_ignoring_files.yaml]: Done!

**Review by alokedesai — APPROVED** [review]
LGTM! 🚀 

Thanks so much @vjindal0112! Looking forward to seeing you soon :)

**Pull request merged**